src/main/webapp/app/modules/games/games.scss

```diff
@@ -1,3 +1,12 @@
+$purple-dark: #6c46da;
+$purple-100: #7f52ff;
+$purple-50: #bfa9ff;
+$purple-30: #d9cbff;
+$purple-15: #e5dcff;
+$grey-100: #27282c;
+$grey-50: #939496;
+$grey-20: #d4d4d5;
+$grey-light: #f6f6f7;
```

Comment:
Hey @mbarzilai, I couldn't figure out a way how to link this .scss file to the app.scss file so I added these colors in here - is there a better way to do this?

Reply:
Per Matti, better way not known